.github/workflows/preview-docs.yml

```diff
@@ -0,0 +1,34 @@
+name: preview docs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
```

Comment:
What's the target use case for running on closed? You might be able to remove this.

Reply:
Based on the action docs I think the close event is used to delete the preview--the build is skipped on line 26 if the event is closed.